sources/HUBViewModelLoaderImplementation.m

```diff
@@ -174,6 +172,8 @@ - (BOOL)isLoading
 
 - (void)loadViewModel
 {
+    [self.connectivityStateResolver addObserver:self];
```

Comment:
@albhaf Could there be any issues with calling `addObserver:` multiple times? As `loadViewModel` is called whenever the view controller appears?